apps/deploy-web/src/hooks/useWhenNot.ts

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+
+export function useWhenNot<T>(condition: T, run: () => void, deps: unknown[] = [], ifNot: () => void): void {
```

Comment:
this is inherently misleading. It's read as `run this function if condition is false`. The last argument just raises questions and you gotta check implementation.

generally you can use multiple calls to `useWhen`

```typescript
const condition: boolean

useWhen(condition, cb)
useWhen(!condition, cb)
```